postmortem/2026-04-23-linux-glibc-baseline.md

```
@@ -0,0 +1,122 @@
+# 2026-04-23 — Linux: glibc 2.38/2.39 not found on current Chromebooks
+
+## What happened
+
+`v0.1.0-beta.34` shipped a Linux tarball at
+`con-0.1.0-beta.34-linux-x86_64.tar.gz`. On a current ChromeOS
+Chromebook (Crostini container, Debian 12 Bookworm, glibc 2.36),
+launching the binary failed with:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)